backend/impl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRepository.java

```diff
@@ -515,7 +516,7 @@ private CreatedAndDuplicatedAndSkippedBranches<NonRootManagedBranchSnapshot> cre
                 "commit (${pointedCommit.getHash().getHashString()}) but parent branch commit " +
                 "is NOT ancestor of parent-agnostic fork point (${parentAgnosticForkPointString}), " +
                 "so we assume that parent-aware fork point = parent branch commit");
-            return ForkPointCommitOfManagedBranch.fallbackToParent(parentPointedCommit);
```

Comment:
Huh for `ForkPointCommitOfManagedBranch` I'd leave the original method, it's pretty confusing now